### ISSUE TITLE
ccleaner@6.29.11342: hash check failed

### ISSUE THREAD

**Issue by ackalker** [ScoopInstaller/Extras#14250]
Transcript of console output:
```
Updating 'ccleaner' (6.28.11297 -> 6.29.11342)
Downloading new version
Starting download with aria2 ...
Download: Download Results:
Download: gid   |stat|avg speed  |path/URI
Download: ======+====+===========+=======================================================
Download: b96a08|OK  |    78MiB/s|C:/Users/xxxxx/scoop/cache/ccleaner#6.29.11342#44855ab.zip
Download: Status Legend:
Download: (OK):download completed.
Get-Content: C:\Users\xxxxx\scoop\apps\scoop\current\lib\core.ps1:1376
Line |
1376 |          return Get-Content $file -AsByteStream -TotalCount 8
     |                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | Operation did not complete successfully because the file contains a
     | virus or potentially unwanted software. :
     | 'C:\Users\xxxxx\scoop\cache\ccleaner#6.29.11342#44855ab.zip'
Checking hash of ccsetup629.zip ... Get-FileHash: C:\Users\xxxxx\scoop\apps\scoop\current\lib\install.ps1:634
Line |
 634 |      $actual = (Get-FileHash -Path $file -Algorithm $algorithm).Hash.T …
     |                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | Operation did not complete successfully because the file contains a
     | virus or potentially unwanted software. :
     | 'C:\Users\xxxxx\scoop\cache\ccleaner#6.29.11342#44855ab.zip'
InvalidOperation: C:\Users\xxxxx\scoop\apps\scoop\current\lib\install.ps1:634
Line |
 634 |      $actual = (Get-FileHash -Path $file -Algorithm $algorithm).Hash.T …
     |      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | You cannot call a method on a null-valued expression.
Get-Content: C:\Users\xxxxx\scoop\apps\scoop\current\lib\core.ps1:1376
Line |
1376 |          return Get-Content $file -AsByteStream -TotalCount 8
     |                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | Operation did not complete successfully because the file contains a
     | virus or potentially unwanted software. :
     | 'C:\Users\xxxxx\scoop\cache\ccleaner#6.29.11342#44855ab.zip'
ERROR Hash check failed!
App:         extras/ccleaner
URL:         https://download.ccleaner.com/portable/ccsetup629.zip
First bytes:
Expected:    3223923938daa84c17ba06e36e7864788e3609cc217642714b559d80452911c2
Actual:

Please try again or create a new issue by using the following link and paste your console output:
https://github.com/ScoopInstaller/Extras/issues/new?title=ccleaner%406.29.11342%3a+hash+check+failed
```

**Comment by ackalker** [comment]
My guess is that the file was blocked by Windows Defender, I was unsure where the error message came from, Windows or Scoop, so I just followed instructions and created this issue.

**Comment by YiHui-Liu** [comment]
yes, i think that it it blocker by the Windows Defender:
```
Line |
 634 |      $actual = (Get-FileHash -Path $file -Algorithm $algorithm).Hash.T …
     |                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | The operation could not be completed successfully because the file contains a virus or potentially unwanted software : 'D:\Softwares\Scoop\cache\ccleaner#6.29.11342#44855ab.zip'
InvalidOperation: D:\Softwares\Scoop\apps\scoop\current\lib\install.ps1:634
Line |
 634 |      $actual = (Get-FileHash -Path $file -Algorithm $algorithm).Hash.T …
     |      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | You cannot call a method on a null-valued expression.
Get-Content: D:\Softwares\Scoop\apps\scoop\current\lib\core.ps1:1376
Line |
1376 |          return Get-Content $file -AsByteStream -TotalCount 8
     |                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | The operation could not be completed successfully because the file contains a virus or potentially unwanted software : 'D:\Softwares\Scoop\cache\ccleaner#6.29.11342#44855ab.zip'
```
